.github/workflows/github-pages.yml

```diff
@@ -7,31 +8,35 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2.3.1 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
-        with:
-          persist-credentials: false
-      - name: NPM Install
+        uses: actions/checkout@v3
+      - name: Yarn Install
         uses: bahmutov/npm-install@v1
```

Comment:
More out of curiosity than anything else (not scope of this PR/review): Why are we using this over e.g., `actions/setup-node` (which has caching built-in and is the "officially recommended" way of setting up a NodeJS env) and then having a

```yml
- name: Yarn Install
  run: yarn install # or whatever specific thing we need
```